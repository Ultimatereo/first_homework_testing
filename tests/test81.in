5
-544257939953 -916026661603 -67476718180 -941266372509 55852244944 737402672748
466500430846 776744279569 -812514941187 -440930665071 899312670837 -530171539021
-882660749832 771468425848 783594036362 470871331856 164526826718 199093873132
32262529205 -37646014302 685671031376 -297246126056 401375031749 -455967906886
906348938048 -185596871944 -591005829590 40396592790 91404972645 -887009637524
