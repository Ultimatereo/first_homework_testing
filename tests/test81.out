-0.633579
0.885048
-0.123066
-1.37706
-1.81165
